src/backoffice/settings/dashboards/examine-management/views/modal-views/fields-viewer.element.ts

```diff
@@ -16,11 +16,7 @@ export class UmbModalLayoutFieldsViewerElement extends UmbModalLayoutElement<Sea
 				display: flex;
 				flex-direction: column;
 				height: 100%;
-				background-color: var(--uui-color-surface);
```

Comment:
What does this change do?

Reply:
Well --uui-color-surface is white, but there was no need to explicitly set the background color to be white, when the parent is white.